src/session.rs

```diff
@@ -91,12 +91,13 @@ impl<FS: Filesystem> Session<FS> {
     pub fn run(&mut self) -> io::Result<()> {
         // Buffer for receiving requests from the kernel. Only one is allocated and
         // it is reused immediately after dispatching to conserve memory and allocations.
-        let mut buffer: Vec<u8> = Vec::with_capacity(BUFFER_SIZE);
+        let mut buffer = vec![0; BUFFER_SIZE];
+        let buf = aligned_sub_buf(buffer.deref_mut(), 8);
```

Comment:
Can you replace the `8` with `align_of::<fuse_in_header>`? I think that will be more clear, and constants like this make me nervous that they will break one day when moving to a new platform -- e.g. u128 is 16 byte aligned.

Reply:
oh, actually I'm wrong. u128 alignment is 8, not 16. On a theoretical platform in the future, it seems like it might be though

Reply:
Done.

Technically it's not `align_of::<fuse_in_header>` we want here. We want the max of the alignments of all the structures we'll be casting to, but in practice this will never be a problem, and what you suggest reads a lot better.